modules/tests/src/smithy4s/tests/PizzaSpec.scala

```diff
@@ -357,24 +401,30 @@ abstract class PizzaSpec
     "X-mIxEd-HeAdEr"
   )
 
+  // note: these aren't really part of the pizza suite
```

Comment:
These tests actually don't exercise the specific backend, but they rely on pizza spec - could possibly be extracted to another spec.

Reply:
nah I'm fine with them for now. We'll do an overhaul of the tests later on